tests/unittests/Foundation/ReferenceFoundation/TestNSURLResponse.mm

```diff
@@ -34,6 +34,7 @@
     NSURLResponse* res =
         [[[NSURLResponse alloc] initWithURL:url MIMEType:@"text/plain" expectedContentLength:0 textEncodingName:nil] autorelease];
     ASSERT_OBJCEQ_MSG(url, res.URL, @"should be the expected url");
+    ASSERT_OBJCEQ(@"path.txt", res.suggestedFilename);
 }
```

Comment:
These three tests should not even test filenames; nor should the ones below. Only the filename tests should test them!

---
In reply to: [95277050](https://github.com/Microsoft/WinObjC/pull/1662#discussion_r95277050) [](ancestors = 95277050,95276334)